packages/graphback-datasync/src/DataSyncPlugin.ts

```diff
@@ -131,7 +131,8 @@ export class DataSyncPlugin extends GraphbackPlugin {
       }
     });
 
-    if (this.config.useVersion) {
+    const modelUsesVersion = !!this.config.modelConfigMap[model.graphqlType.name]?.enabled;
```

Comment:
I don't think a double-negation is required here.

```suggestion
    const modelUsesVersion = !this.config.modelConfigMap[model.graphqlType.name]?.enabled;
```

Reply:
Hmm wouldn't that make the expression true when the rest of the statement is undefined?

Reply:
My bad, what I mean to say I don't think there is a need for **any** negation here.